lib/host.js

```diff
@@ -63,11 +66,11 @@ Host.prototype.setDown = function() {
     //This host is not down, the pool is being shutdown
     return;
   }
-  this.log('warning', 'Setting host ' + this.address + ' as DOWN');
-  this.unhealthyAt = new Date().getTime();
   this.reconnectionDelay = this.reconnectionSchedule.next().value;
+  this.log('warning', util.format('Host %s considered as DOWN. Reconnection delay %dms.', this.address, this.reconnectionDelay));
+  this.unhealthyAt = new Date().getTime();
```

Comment:
I think using `canBeConsideredAsUp()` makes no sense now as it should be only one reconnection attempt scheduled and is not being considered by the lbp in the mean time.
`is it up? -> set it down`
I'll replace it by calling `isUp()` instead. `canBeConsideredAsUp()` should remain there as it is part of the API.